dev/.env.docker-compose

```diff
@@ -5,4 +5,4 @@ DJANGO_MINIO_STORAGE_ENDPOINT=minio:9000
 DJANGO_MINIO_STORAGE_ACCESS_KEY=minioAccessKey
 DJANGO_MINIO_STORAGE_SECRET_KEY=minioSecretKey
 DJANGO_STORAGE_BUCKET_NAME=django-storage
-DJANGO_MINIO_STORAGE_MEDIA_URL=http://localhost:9000/django-storage
+DJANGO_MINIO_STORAGE_MEDIA_URL=http://minio:9000/django-storage
```

Comment:
I'm not sure this is actually desired.  Specifically, `minio` is the host name within the docker network, but won't work outside of the docker network (e.g., from a browser on localhost), whereas if the port is forwarded, it will work on localhost (but not from an external machine).

Reply:
In my deployment, I've got ` DJANGO_MINIO_STORAGE_ENDPOINT=<hostname of my machine>:<port>`, where I've used the actual host name on our VPN.  This lets me access the application from another machine in or vpn and still upload and download to minio.

Reply:
I see. I did not realize this is for manufacturing the URL for clients.

Reply:
I could be wrong, as I might be confusing settings we had earlier on in the project.  Did you check if uploads and downloads both work from the local machine with `minio` in the url?

Reply:
I just checked and can confirm that the change I proposed here would break the uploads/downloads. For my future reference, the `DJANGO_MINIO_STORAGE_MEDIA_URL` environment variable [is used to set the "settings.py" variable `MINIO_STORAGE_MEDIA_URL`](https://github.com/girder/django-composed-configuration/blob/8c210a44046c691f28c358b4f3efe868e0a2b03b/composed_configuration/_configuration.py#L54-L56) for the package [django-minio-storage](https://github.com/py-pa/django-minio-storage). The setting is documented [here](https://django-minio-storage.readthedocs.io/en/stable/usage/). This variable is used to create the URLs provided to clients for files that live in minio. It's analogous to the [stock Django `MEDIA_URL` setting](https://docs.djangoproject.com/en/3.1/ref/settings/#media-url).